src/app-layer-modbus.h

```diff
@@ -34,25 +34,6 @@
 #ifndef __APP_LAYER_MODBUS_H__
 #define __APP_LAYER_MODBUS_H__
 
-#include "decode.h"
-#include "detect-engine-state.h"
-#include "queue.h"
-#include "rust.h"
-
-/* Modbus Application Data Unit (ADU)
- * and Protocol Data Unit (PDU) messages */
-enum {
-    MODBUS_DECODER_EVENT_INVALID_PROTOCOL_ID,
-    MODBUS_DECODER_EVENT_UNSOLICITED_RESPONSE,
-    MODBUS_DECODER_EVENT_INVALID_LENGTH,
-    MODBUS_DECODER_EVENT_INVALID_UNIT_IDENTIFIER,
-    MODBUS_DECODER_EVENT_INVALID_FUNCTION_CODE,
-    MODBUS_DECODER_EVENT_INVALID_VALUE,
-    MODBUS_DECODER_EVENT_INVALID_EXCEPTION_CODE,
-    MODBUS_DECODER_EVENT_VALUE_MISMATCH,
-    MODBUS_DECODER_EVENT_FLOODED,
-};
-
 /* Modbus Function Code Categories. */
 #define MODBUS_CAT_NONE                 0x0
```

Comment:
These seem unused and could be removed, right ?

Reply:
Yes, good find. I think they popped back up during a rebase.